lib/models/task.js

```diff
@@ -6,6 +6,17 @@ class Task extends CoreObject {
   run(/*options*/) {
     throw new Error('Task needs to have run() defined.');
   }
+
+  /**
+   * Interrupt comamd with an exit code
+   * Called when the process is interrupted from outside, e.g. CTRL+C or `process.kill()`
+   *
+   * @private
+   * @method onInterrupt
+   */
+  onInterrupt() {
+    process.exit(1);
```

Comment:
> Otherwise seems we need some other out of band way to communicate exit status

for zero and one it's obviously pretty easy, if we actually feel like we need more we could just add an `exitCode` property to the `Error` instance and use that instead if it exists.